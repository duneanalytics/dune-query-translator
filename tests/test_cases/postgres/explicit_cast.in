select cast(x as int), y::text, cast(c + 1 as int) from t
