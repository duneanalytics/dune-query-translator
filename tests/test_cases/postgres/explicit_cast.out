/* Success! If you're still running into issues, check out https://dune.com/docs/query/syntax-differences/ or reach out in the #dune-sql Discord channel. */

SELECT
  TRY_CAST(x AS INTEGER) AS x,
  TRY_CAST(y AS VARCHAR) AS y,
  TRY_CAST(c + 1 AS INTEGER)
FROM t